data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/command/dataimport/ImportCommandOptions.java

```diff
@@ -39,6 +42,15 @@ public class ImportCommandOptions {
       defaultValue = "16")
   protected int maxThreads;
 
+  // Deprecated option - kept for backward compatibility
+  @CommandLine.Option(
+      names = {DEPRECATED_THREADS_OPTION},
+      paramLabel = "<THREADS>",
+      description = "Deprecated: Use --max-threads instead",
```

Comment:
Just a question, but why `threads` are renamed to `max-threads`?
It creates threads dynamically within the max, so that the number could be lower than the max depending on workloads?

Reply:
@brfrn169 That is probably best as it will involve renaming some variables in the core code as well so it's better done separately.  Feel free to merge this one, and I will push the other PR tomorrow morning. (I already have the code ready)